src/Moryx.AbstractionLayer.Products.Endpoints/WorkplanVersionsController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moryx.Workflows;
+using System.Collections.Generic;
+
+namespace Moryx.AbstractionLayer.Products.Endpoints
+{
+    /// Definition of a REST API on the <see cref="IWorkplansVersions"/> facade.
```

Comment:
Do we want to call it the WorkplanVersionsController or just WorkplansController? 😅